rajawali/src/main/java/org/rajawali3d/postprocessing/passes/EffectPass.java

```diff
@@ -24,66 +24,63 @@
-		mEnabled = true;
-		mRenderToScreen = false;
-	}
+    public EffectPass() {
+        mPassType = PassType.EFFECT;
+        mNeedsSwap = true;
+        mClear = false;
+        mEnabled = true;
+        mRenderToScreen = false;
+    }
 
-	public EffectPass(Material material) {
-		this();
-		setMaterial(material);
-	}
+    public EffectPass(Material material) {
+        this();
+        setMaterial(material);
+    }
 
-	protected void createMaterial(@NonNull VertexShader vertexShader, @NonNull FragmentShader fragmentShader)
-	{
-		mVertexShader = vertexShader;
-		mFragmentShader = fragmentShader;
-		mVertexShader.setNeedsBuild(false);
-		mFragmentShader.setNeedsBuild(false);
-		setMaterial(new Material(mVertexShader, mFragmentShader));
-	}
+    protected void createMaterial(@NonNull VertexShader vertexShader, @NonNull FragmentShader fragmentShader) {
+        mVertexShader = vertexShader;
+        mFragmentShader = fragmentShader;
+        mVertexShader.setNeedsBuild(false);
+        mFragmentShader.setNeedsBuild(false);
+        setMaterial(new Material(mVertexShader, mFragmentShader));
+    }
 
-	protected void createMaterial(int vertexShaderResourceId, int fragmentShaderResourceId)
-	{
-		createMaterial(new VertexShader(vertexShaderResourceId), new FragmentShader(fragmentShaderResourceId));
-	}
+    protected void createMaterial(int vertexShaderResourceId, int fragmentShaderResourceId) {
+        createMaterial(new VertexShader(vertexShaderResourceId), new FragmentShader(fragmentShaderResourceId));
+    }
 
 
-	public void setShaderParams()
-	{
-		mFragmentShader.setUniform1f(PARAM_OPACITY, mOpacity);
-		mMaterial.bindTextureByName(PARAM_TEXTURE, 0, mReadTarget.getTexture());
-	}
+    public void setShaderParams() {
+        mFragmentShader.setUniform1f(PARAM_OPACITY, mOpacity);
+        mMaterial.bindTextureByName(PARAM_TEXTURE, 0, mReadTarget.getTexture());
+    }
 
-	public void render(Scene scene, Renderer renderer, ScreenQuad screenQuad, RenderTarget writeTarget, RenderTarget readTarget, long ellapsedTime, double deltaTime) {
-		mReadTarget = readTarget;
-		mWriteTarget = writeTarget;
-		screenQuad.setMaterial(mMaterial);
-		screenQuad.setEffectPass(this);
+    public void render(Scene scene, Renderer renderer, ScreenQuad screenQuad, RenderTarget writeTarget,
+                       RenderTarget readTarget, long ellapsedTime, double deltaTime) {
+        mReadTarget = readTarget;
+        mWriteTarget = writeTarget;
+        screenQuad.setMaterial(mMaterial);
+        screenQuad.setEffectPass(this);
 
-		if(mRenderToScreen)
-			scene.render(ellapsedTime, deltaTime, null);
-		else
-			scene.render(ellapsedTime, deltaTime, writeTarget);
-	}
+        if (mRenderToScreen) {
+            scene.render(ellapsedTime, deltaTime, null);
+        } else {
+            scene.render(ellapsedTime, deltaTime, writeTarget);
+        }
+    }
 
-	public void setOpacity(float value)
-	{
-		mOpacity = value;
-	}
+    public void setOpacity(float value) {
```

Comment:
Similar to NonNull, would be good to have a @ FloatRange(from=0f,to=1f) annotation here.